exercises/exercise-11/declarations/stats/index.d.ts

```diff
@@ -1,3 +1,16 @@
 declare module 'stats' {
-    export function getMaxIndex(input: unknown, comparator: unknown): unknown;
+    type ComparatorFn<T, R> = (input: T[], comparator: (a: T, b: T) => number) => R;
+
+    type CompArgs<T> = [T[], (a: T, b: T) => number];
+
+    function GetIndexFn<T>(input: T[], comparator: (a: T, b: T) => number): number;
+    function GetElementFn<T>(input: T[], comparator: (a: T, b: T) => number): T | null;
+
+    export const getMaxIndex: typeof GetIndexFn;
```

Comment:
It's interesting that you can use `typeof Fn` here to "inherit" the generic parameter. I don't think you can do such a thing with a type alias. (I tried above.)